scripts/entrypoint.sh

```diff
@@ -3,5 +3,5 @@ set +ex
 
 ecosystem_list=("openshift" "knative" "kubevirt")
 
-# Run the data collector for given eco systems and no of days to colelct data
-python3 app/run_data_collector.py -eco-systems ${ecosystem_list[*]} -days ${DAYS};
+# Run the data collector for given eco systems and no of days to  collect data
+python3 run_data_collector.py -e ${ecosystem_list[*]} -d ${DAYS};
```

Comment:
It is better to specify absolute path of this script here.

Reply:
we are copying run_data_collector.py to root path only and using the same.